src/react-contenteditable.js

```diff
@@ -46,11 +54,14 @@ export default class ContentEditable extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    this.observer.disconnect();
+  }
+
   emitChange(evt) {
     if (!this.htmlEl) return;
     var html = this.htmlEl.innerHTML;
     if (this.props.onChange && html !== this.lastHtml) {
-      evt.target = { value: html };
       this.props.onChange(evt);
```

Comment:
Isn't it enough to get the whole MutationRecord (=`evt`) in onChange? The element's `innerHtml` is then accessible via `evt.target.innerHtml`.